Plugins~/Src/MeshSyncClientBlender/msblenContextGeometryNodes.cpp

```diff
@@ -72,19 +72,19 @@ void msblenContext::exportInstances() {
 }
 void msblenContext::exportInstancesFromFile(Object* instancedObject, Object* parent, SharedVector<mu::float4x4> mat, mu::float4x4& inverse)
 {
-    mu::parallel_for(0, mat.size(), 10, [this, &mat, &inverse](int i)
+    mu::parallel_for(0, mat.size(), 10, [this, &instancedObject, &mat, &inverse](int i)
```

Comment:
`inverse` is not used anymore. We can probably remove it ?

Reply:
It's used again now.